cmd/swapd/main.go

```diff
@@ -76,6 +75,7 @@ const (
 	flagGasPrice             = "gas-price"
 	flagGasLimit             = "gas-limit"
 	flagUseExternalSigner    = "external-signer"
+	flagClaimRelayer         = "claim-relayer"
```

Comment:
not really a fan of this flag, I'd just call it `relayer` or `is-relayer` or something?

Reply:
I'll change it to `relayer`

Reply:
Renamed